docs/source/installation.rst

```diff
@@ -2,152 +2,133 @@ Installation
 ============
 
 Supported OS
-------------
+++++++++++++
 
-Currently, we only support ``MacOS`` and ``Linux`` operating systems. ``Windows``
-users have the possibility to install and use ``ivadomed`` via
-`Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/>`_. The steps below (about updating bashrc) are strongly recommended for MacOS users in the installation process but are optional for Linux users.
+    Currently, ``ivadomed`` supports GPU/CPU on ``Linux`` and ``Windows``, and CPU only on ``macOS`` and `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_.
 
-Open your bash/zsh script file with editor on your computer.
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using bash shell
-    ::
+    You can setup ``ivadomed`` using either Conda or Venv:
 
-        vim ~/.bashrc
+    .. tabs::
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+        .. tab:: Install via ``venv``
 
-Write in your .bashrc/.zshrc file with following line.
+            1. Setup Python Venv Virtual Environment.
 
-::
+                ``ivadomed`` requires Python >= 3.6 and <3.10.
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+                First, make sure that a compatible version of Python 3 is installed on your system by running:
 
-Save this change and restart your terminal to apply the change.
+                .. code::
 
-There are fundamentally two different approaches to install ``IvadoMed``:
+                   python3 --version
 
-1) Install via Conda
-    This is the easiest way for personal computers.
+                If your system's Python is not 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all), please `install Python <https://realpython.com/installing-python/>`_ before continuing.
```

Comment:
```suggestion
                If your system's Python is not 3.6, 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all), please `install Python <https://realpython.com/installing-python/>`_ before continuing.
```

Reply:
Oh yeah, good point. Not sure why we dropped 3.6 support, even though we technically still test them.